models/documentlistmodel.h

```diff
@@ -62,7 +62,8 @@ class DocumentListModel : public QAbstractListModel
     virtual QHash<int, QByteArray> roleNames() const;
 
     void setAllItemsDirty(bool status);
-    void addItem(QString name, QString path, QString type, int size, QDateTime lastRead, QString mimeType);
+    void addItem(const QString &name, const QString &path, const QString &type, int size, QDateTime date,
```

Comment:
Just wondering out loud: could `date` be also const reference?

Reply:
QDateTime instances should be small so the cost of the reference should be bigger than making a copy so left it this way on purpose.